java/src/main/java/com/saucelabs/saucebindings/SauceSession.java

```diff
@@ -11,32 +13,46 @@
 import java.net.URL;
 
 public class SauceSession {
-    @Getter @Setter private DataCenter dataCenter = DataCenter.US_WEST;
-    @Getter private final SauceOptions sauceOptions;
+    @Getter @Setter protected DataCenter dataCenter = DataCenter.US_LEGACY;
+    @Getter private SauceOptions sauceOptions;
     @Setter private URL sauceUrl;
-
-    @Getter private RemoteWebDriver driver;
+    @Getter private RemoteWebDriver webDriver;
+    @Getter private AppiumDriver appDriver;
 
     public SauceSession() {
         this(new SauceOptions());
     }
 
     public SauceSession(SauceOptions options) {
-        sauceOptions = options;
+        this.sauceOptions = options;
     }
 
     public RemoteWebDriver start() {
-        driver = createRemoteWebDriver(getSauceUrl(), sauceOptions.toCapabilities());
-        return driver;
+        String environment = sauceOptions.toCapabilities().getCapability("platformName").toString();
+        String browserName = sauceOptions.toCapabilities().getBrowserName();
+
+        if (browserName.equals("")){
+            if (environment.toLowerCase().equals("android")){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using equalsIgnoreCase() is cleaner than using toUpperCase/toLowerCase().equals().](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=5346038)